docs/howto/logging.txt

```diff
@@ -117,11 +117,11 @@ file ``general.log`` (at the project root):
     :emphasize-lines: 3-8
 
     LOGGING = {
-        [...]
-        'handlers': {
-            'file': {
-                'class': 'logging.FileHandler',
-                'filename': 'general.log',
+        # [...]
```

Comment:
As far as I'm aware we want to keep the `...` and `…` unchanged in docs.

Reply:
If I try to change this, since it's `code-block:: python` it gives these errors:

```
../docs/howto/logging.txt:116: code block parse error Cannot parse: 3:4:     "handlers": {
../docs/howto/logging.txt:155: code block parse error Cannot parse: 3:4:     "loggers": {
../docs/howto/logging.txt:194: code block parse error Cannot parse: 3:4:     "formatters": {
../docs/howto/logging.txt:220: code block parse error Cannot parse: 3:4:     "handlers": {
../docs/howto/logging.txt:255: code block parse error Cannot parse: 3:4:     "loggers": {
../docs/howto/logging.txt:269: code block parse error Cannot parse: 3:4:     "loggers": {
```

I.e. it fails to interpret the lines containing `...`. Unlike `code-block:: pycon`, the [PythonConsoleLexer](https://pygments.org/docs/lexers/#pygments.lexers.python.PythonConsoleLexer), which does. But that's not meant for Python code. Not sure what direction to take on this. Will change `# [...]` to `# ...` and see if that's acceptable.